doc/ocp_lock/lock_spec.ocp

```diff
@@ -519,7 +519,7 @@ At initial boot, Caliptra firmware performs the following steps:
 - Determines whether a HEK is available, based on the fuse configuration. If a HEK is unavailable, KMB enters a mode where MEKs are not allowed to be loaded.
 - Initializes random HPKE keypairs for each supported algorithm, and assigns handles to them, reported via [ENUMERATE_KEM_HANDLES](#sec:enumerate-kem-handles).
 
-The Ready MPK encryption key is not initialized at boot-up, as it is generated lazily upon first use. See @sec:mpks for details.
+The Enabled MPK encryption key is not initialized at boot-up, as it is generated lazily upon first use. See @sec:mpks for details.
```

Comment:
Other references to this key have all words capitalized. We should probably do the same here.
```suggestion
The Enabled MPK Encryption Key is not initialized at boot-up, as it is generated lazily upon first use. See @sec:mpks for details.
```